src/z3c/saconfig/README.rst

```diff
@@ -333,41 +334,41 @@ Configuration using ZCML
 A configuration directive is provided to register a database engine
 factory using ZCML.
 
-  >>> from cStringIO import StringIO
+  >>> from io import BytesIO
   >>> from zope.configuration import xmlconfig
   >>> import z3c.saconfig
   >>> xmlconfig.XMLConfig('meta.zcml', z3c.saconfig)()
 
 Let's try registering the directory again.
 
-  >>> xmlconfig.xmlconfig(StringIO("""
+  >>> xmlconfig.xmlconfig(BytesIO(six.b("""
```

Comment:
No need for `six.b`, `b"..."` literals work across all currently supported Python versions.